### DISCUSSION TITLE
MultiSelect with value contains comma (,)

### DISCUSSION THREAD

**Discussion by arielshulman** [mantinedev/mantine#3533] [Q&A]
It seems that if I have a value in `MultiSelect` which contains comma I won't be able to extract it back correctly from `value` with split.
Any idea how to handle this?

I think it related to this raw:
https://github.com/mantinedev/mantine/blob/f4857b6e90b2a8173f9cfa2c893b900ef15e1cc9/src/mantine-core/src/components/MultiSelect/MultiSelect.tsx#L558
Thanks 🙏

**Comment by rtivital** [comment]
You can provide values that do not include comas. For example, transform `,` to `|` before passing to the component, then transform back after splitting with comas.
